modules/odc-interacting-with-applications-and-components.adoc

```diff
@@ -5,8 +5,9 @@
 [id="odc-interacting-with-applications-and-components_{context}"]
 = Interacting with the application and the components
 
-The *Topology* view in the *Developer* perspective of the web console provides the following options to interact with the application and the components:
+The *Topology View* in the Developer perspective of the web console provides the following options to interact with the application and the components:
```

Comment:
s/with the application and the components/with applications and components

Reply:
Done